packages/viz/src/lib/style/helpers/math-utils.ts

```diff
@@ -0,0 +1,11 @@
+const lerp = (x: number, y: number, a: number) => x * (1 - a) + y * a;
+const clamp = (a: number, min = 0, max = 1) => Math.min(max, Math.max(min, a));
+export const invlerp = (x: number, y: number, a: number) =>
+  clamp((a - x) / (y - x));
+export const range = (
```

Comment:
I understand `clamp` and `invlerp`, but what's `range` in this context ?

Reply:
It's the interpolation between two ranges, it onverts a value from one data range to another

Reply:
Ok, why don't we give it a more explicit name and a header?